src/Tizen.NUI.Components/Controls/Pagination.cs

```diff
@@ -512,18 +507,26 @@ public override void OnInitialize()
             AccessibilityHighlightable = true;
             AppendAccessibilityAttribute("style", "pagecontrolbyvalue");
 
-            container = new VisualView()
+            container = new View()
             {
                 Name = "Container",
                 ParentOrigin = Tizen.NUI.ParentOrigin.CenterLeft,
                 PivotPoint = Tizen.NUI.PivotPoint.CenterLeft,
                 PositionUsesPivotPoint = true,
+                Layout = new LinearLayout()
+                {
+                    LinearOrientation = LinearLayout.Orientation.Horizontal,
+#pragma warning disable CA2000
+                    CellPadding = new Size(indicatorSpacing, 0),
```

Comment:
The type of CellPadding is Size2D.
Is this related to CA2000?

Reply:
Actually, no. It's not related to CA2000. The warning message is to call Dispose() or use `using` statement instead.
But, I'll remove warning suppression once.

Reply:
Oh, the error message is because of the type mismatching! 
Thank you for checking it!! :+1: